test/Mono.Linker.Tests.Cases/Reflection/RuntimeReflectionExtensionsCalls.cs

```diff
@@ -1,9 +1,11 @@
 using System;
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
 using System.Reflection;
+using System.Runtime.CompilerServices;
 
 namespace Mono.Linker.Tests.Cases.Reflection
 {
+	[SkipKeptItemsValidation]
```

Comment:
I think we should leave the existing tests working via the Kept attributes. We need both kinds of tests - the more "End-2-end" which validate that linker correctly marks the members. Some of the data flow tests go without the Kept attribute as they are mostly validating that we correctly report warnings, in which case validating what was kept is not that interesting.

I think that to test this change we should stick to the Kept-style testing.